### DISCUSSION TITLE
posts/2023/11/firefox-brink/

### DISCUSSION THREAD

**Discussion by giscus** [brycewray/comments#41] [Announcements]
# posts/2023/11/firefox-brink/

The Big Three may effectively be down to a Big Two, and right quick.

https://www.brycewray.com/posts/2023/11/firefox-brink/?utm_source=changelog-news

**Comment by neontuna** [comment]
Hopefully unlike IE supporting or not supporting Firefox isn't such a binary.  IE support required all kinds of nonsense that we're all thankful is behind us finally.  Supporting Firefox just means not using some feature thats exclusive to the other engines - maybe a tall order in the future but doesn't seem too difficult at present.

**Reply by brycewray** [reply]
@neontuna I agree with your sentiment but, having worked both with and in corporate IT departments, I’m not as optimistic. We’ve already seen too many reports in the last two or three years about how certain financial sites refuse to accept Firefox, so I can readily believe the situation will get worse.

**Comment by wistlo** [comment]
Firefox and Edge are the two approved browsers at the big telecom I worked at earlier this year, but most new applications are coming from outside vendors and they show a definite preference for Chrome or Chromium-based clients.  

Users seemed reluctant to even try Firefox even though it was already up and running on their desktop image.  Ironically, for the Oracle Siebel application that always seemed to kick Edge into bothersome "IE Compatibility" mode, Firefox turned out to be faster and more stable, but that is a legacy app that is scheduled to be retired by 2018 (they're running a lil behind on it).   

I notice Debian with Gnome still installs with Firefox as default, and I continue to use it there. Fortunately the UI differences among browsers (at least in Windows) have diminished, so the remembering different sets keystrokes bit from years ago is mostly gone.  

We should all have qualms about Google or Microsoft (or in the case of Edge, both) completely owning the gateway into every user interface.  I hope Firefox sticks around.

**Comment by DeclanChidlow** [comment]
I got a lot out of your article. It was fantastic.

I ended up referencing it in an article I wrote about the dominance of Chrome on the web.
https://vale.rocks/posts/everything-is-chrome

It certainly is scary to think what could come in the very near future.

Cheers

**Reply by brycewray** [reply]
@DeclanChidlow Thanks for the kind words, the reference, **and** — not the least — your own article! And, yes, the potential near-future experience of web browsing is troubling given that, as the saying goes, “Those who cannot remember the past are doomed to repeat it,” and there would seem to be too many folks in high places who have all-too-short memories. 😞
